frontend/src/components/ui/heading/Heading.svelte

```diff
@@ -0,0 +1,13 @@
+<div class="fixed top-0 w-full border-b-[1px] border-gray-400">
```

Comment:
`border-b` achieves the same effect as `border-b-[1px]`. Usage of arbitary values e.g. `text-[#8f9a9f]` `border-[4px]` is considered bad practice unless in unavoidable situations.